tests/playwright/src/utility/fixtures.ts

```diff
@@ -30,21 +30,29 @@ export type TestFixtures = {
   navigationBar: NavigationBar;
   welcomePage: WelcomePage;
   page: Page;
+  context: BrowserContext;
   statusBar: StatusBar;
 };
 
 export type FixtureOptions = {
   runnerOptions: RunnerOptions;
 };
 
+let newContext = false;
+
 export const test = base.extend<TestFixtures & FixtureOptions>({
   runnerOptions: [new RunnerOptions(), { option: true }],
   runner: async ({ runnerOptions }, use) => {
     const runner = await Runner.getInstance({ runnerOptions });
     await use(runner);
   },
-  page: async ({ runner }, use) => {
-    await use(runner.getPage());
+  context: async ({ runner }, use) => {
+    newContext = true;
+    await use(runner.getNewContext());
+  },
```

Comment:
why does this need to be a fixture, I'm not sure where you are actually using it, but it does not need to be a fixture, you can just can use the `runner` fixture and call the `context()` method exposed by the PW API to use whatever BroserContext is active.